service/storage_proxy.cc

```diff
@@ -2785,7 +2785,12 @@ void storage_proxy::send_to_live_endpoints(storage_proxy::response_id_type respo
                 ++stats.read_repair_write_attempts.get_ep_stat(coordinator);
             }
 
-            if (coordinator == my_address) {
+            if (handler.get_schema()->cf_name() == "tbl_noop") {
```

Comment:
We can use the error injection framework for injection errors.

Reply:
Yes, those are hacks I included only for demonstration purposes of the RFC. I agree we shouldn't leave them in the final version. I sent a separate PR which adds flags to `perf_simple_query` so that you can trigger timeouts and measure their performance without such hacks.

The PR is here: https://github.com/scylladb/scylla/pull/10899